docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+# Compose file for launching a local windbreaker setup
+
+version: '3'
+services:
+  windbreaker:
+    build:
+      context: ./repos/windbreaker
+    depends_on:
+      - dependencies
+    environment:
+      SERVICE_ENVIRONMENT: docker
+    volumes:
+      - ./repos/windbreaker:/usr/windbreaker
+      - /usr/windbreaker/node_modules
+      # uncomment below line as needed to "link" in a local version
+      # the service-util repo. The initial pull will grab it for you
+      #  - ../windbreaker-service-util:/usr/windbreaker/node_modules/windbreaker-service-util
+    command: npm start
+  rss-watcher:
+    build:
+      context: ./repos/rss-watcher
+    depends_on:
+      - dependencies
+    volumes:
+      - ./repos/rss-watcher:/usr/rss-watcher
+      - /usr/rss-watcher/node_modules
+      #  - ../windbreaker-service-util:/usr/rss-watcher/node_modules/windbreaker-service-util
+    command: npm start
+  npm-watcher:
+    build:
+      context: ./repos/npm-watcher
+    depends_on:
+      - dependencies
+    volumes:
+      - ./repos/npm-watcher:/usr/npm-watcher
+      - /usr/npm-watcher/node_modules
+      #  - ../windbreaker-service-util:/usr/npm-watcher/node_modules/windbreaker-service-util
+    environment:
+      SERVICE_ENVIRONMENT: docker
+    command: npm start
+
+
+  # a "grouping" container that is here to keep the
+  # compose file a little more dry
+  dependencies:
+    image: tianon/true
```

Comment:
Super tiny container just to group dependencies together. As we add more services and this docker file will get filled with services containing a lot of the same docker service configuration. It's only 125 bytes and exits out immediately, so I figured it's not a big deal.